.gitignore

```diff
@@ -22,3 +22,4 @@ coverage
 front-end/src/utils/translations/*.json
 test-data
 .vendor-new/*
+package-lock.json
```

Comment:
Dont exlcude this. We need this checked in and tracked for reproducible builds

Reply:
Done. Thank you.

Regards,

José